content/en/developers/faq/data-collection-resolution-retention.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Datadog data collection, resolution, and retention
+kind: faq
+---
+
+Find below a summary of Datadog data collection, resolution, and retention:
+
+| Source                       | Collection Methods                                                    | Collection interval | Minimum Resolution | Default Retention                  | Product Category |
+| ----                         | ----                                                                  | ----                | ----               | ----                               | ----             |
+| System metrics               | Datadog Agent                                                         | 15 secs             | 1 s                | 15 months                          | Infrastructure   |
+| Agent integrations           | Datadog Agent + enabled integrations                                  | 15 secs             | 1 s                | 15 months                          | Infrastructure   |
+| Live Containers              | Datadog Agent + enabled Docker integration OR Datadog container Agent | 2 secs              | 1 s                | 36 hours                           | Infrastructure   |
+| Live Processes               | Datadog Agent + Process Agent                                         | 2 secs              | 1 s                | 36 hours                           | Infrastructure   |
+| Custom Metrics (Agent check) | Datadog Agent + custom Agent check                                    | 15 secs             | 1 s                | 15 months                          | Infrastructure   |
+| Custom Metrics (StatsD)      | Datadog Agent (built-in statsD collector)                             | 15 secs             | 1 s                | 15 months                          | Infrastructure   |
+| Custom Metrics (API)         | POST directly to Datadog's API                                        | Real time           | 1 s                | 15 months                          | Infrastructure   |
+| Events                       | Datadog Agent OR integrations OR API                                  | Real time           | 1 s                | 15 months                          | Infrastructure   |
+| Traces (Sampled)             | Datadog Agent + tracing library                                       | Real time           | 1 micro sec        | See traces retention documentation | APM              |
```

Comment:
Is there a link for the `traces retention documentation`?

Reply:
yup, fixed :)